docker/build-and-push.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Usage: $ ./build-tag-push.sh <version>
+
+set -eu
+
+readonly SEMVER_REGEX=^[0-9]+\.[0-9]+\.[0-9]+$ # X.Y.Z
+readonly ORG=projectmarquez
+readonly REPO=marquez
+
+project_root=$(git rev-parse --show-toplevel)
+cd $project_root
+
+# Version of Marquez image to build
+version=$1
+
+if [[ ! $version =~ $SEMVER_REGEX ]]; then
+  echo "ERROR: Version must match ${SEMVER_REGEX}"
+  exit 1
+fi
+
+echo "Building image (tag: ${version})..."
+
+docker build --no-cache -t ${ORG}/${REPO}:${version} .
+docker tag ${ORG}/${REPO}:${version} ${ORG}/${REPO}:latest
+docker push ${ORG}/${REPO}:${version}
+docker push ${ORG}/${REPO}:latest
+
+echo "DONE!"
```

Comment:
Ran it through a shell script checker (ShellCheck) and it had the following feedback:
```
$ shellcheck myscript
 
Line 5:
 readonly SEMVER_REGEX=^[0-9]+\.[0-9]+\.[0-9]+$ # X.Y.Z
                       ^-- SC2125: Brace expansions and globs are literal in assignments. Quote it or use an array.
 
Line 9:
cd $project_root
   ^-- SC2086: Double quote to prevent globbing and word splitting.
 
Line 17:
 docker build --no-cache -t ${ORG}/${REPO}:${version} .
                                           ^-- SC2086: Double quote to prevent globbing and word splitting.
 
Line 18:
docker tag ${ORG}/${REPO}:${version} ${ORG}/${REPO}:latest
                          ^-- SC2086: Double quote to prevent globbing and word splitting.
 
Line 19:
docker push ${ORG}/${REPO}:${version}
                           ^-- SC2086: Double quote to prevent globbing and word splitting.
```